website/source/api/secret/identity/entity-alias.html.md

```diff
@@ -0,0 +1,219 @@
+---
+layout: "api"
+page_title: "Identity Secret Backend - HTTP API"
+sidebar_current: "docs-http-secret-identity-entity-alias"
+description: |-
+  This is the API documentation for managing entity aliases in the identity store.
+---
+
+## Create an Entity Alias
+
+This endpoint creates a new alias for an entity.
+
+| Method   | Path                       | Produces               |
+| :------- | :------------------------- | :----------------------|
+| `POST`   | `/identity/entity-alias`   | `200 application/json` |
+
+### Parameters
+
+- `name` `(string: <required>)` - Name of the alias. Name should be the identifier
+  of the client in the authentication source. For example, if the alias belongs
+  to userpass backend, the name should be a valid username within userpass
+  backend. If alias belongs to GitHub, it should be the GitHub username.
+
+- `id` `(string: <optional>)` - ID of the entity alias. If set, updates the
+  corresponding entity alias.
+
+- `canonical_id` `(string: <required>)` - Entity ID to which this alias belongs to.
+
+- `mount_accessor` `(string: <required>)` - Accessor of the mount to which the
+  alias should belong to.
+
+- `metadata` `(key-value-map: {})` – Metadata to be associated with the
+  alias.
+
+### Sample Payload
+
+```
+{
+	"name": "testuser",
+	"metadata": {"group=san_francisco", "region=west"},
```

Comment:
Is there a reason why the actual key-value pairs are stringified? 

```json
{
  "metadata": {
    "group": "san_francisco",
    "region": "west"
  }
}
```

Reply:
I think this is invalid JSON syntax. It has to be either an array of strings or like the above.